pkg/cloud/powervs.go

```diff
@@ -260,16 +269,16 @@ func (p *powerVSCloud) CreateDisk(volumeName string, diskOptions *DiskOptions) (
 }
 
 func (p *powerVSCloud) DeleteDisk(volumeID string) (success bool, err error) {
-	err = p.volClient.DeleteVolume(volumeID, p.cloudInstanceID, TIMEOUT)
+	err = p.volClient.DeleteVolume(volumeID)
 	if err != nil {
 		return false, err
 	}
 
 	return true, nil
 }
 
-func (p *powerVSCloud) AttachDisk(volumeID string, nodeID string) (err error) {
-	_, err = p.volClient.Attach(nodeID, volumeID, p.cloudInstanceID, TIMEOUT)
+func (p *powerVSCloud) AttachDisk(volumeName string, volumeID string, nodeID string) (err error) {
+	err = p.volClient.Attach(nodeID, volumeName)
```

Comment:
I know its out of context but any idea why we need both id and name for the attach api?

cc @yussufsh

Reply:
It's vm instance id 1st param and volume name or id for the 2nd.

Reply:
thanks @yussufsh for the information, @Madhan-SWE  can we just use the volumeID to the `Attach` api, that way we don't need to our AttachDisk api

Reply:
`p.volClient.Attach` Requires volume Name as an argument.
Also in our AttachDisk method we use `WaitForVolumeState` that requires `volumeId` as an argument(it internally uses `volClient.Get(volumeID)`).

Hence, we need both volumeId and volumeName in our attach disk method.

Reply:
@yussufsh the second param for `Attach` could be wither volumeName or volumeId ?